README.md

```diff
@@ -4,6 +4,16 @@
 
 ## How does it work?
 
+Please refer to the [official documentation](https://docs.newrelic.com/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection) to learn the reasoning behind this project.
+
+## Setup
+
+Please refer to the [setup instructions in the official documentation](https://docs.newrelic.com/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection#install).
+
+## Development
```

Comment:
IMHO this README is not focused on development. The main difference is that the target would most probably be devs.

Having said that, I want to suggest to avoid making any difference between steps ("How does it work", "Setup"). Instead, just add those "Please refer to..." at the end of each section.

Reply:
I'll do this. But I prefer to put the link to the public docs in the beginning of each section. Because close to the link there will be more in-depth documentation and putting the link after all this explanation feels like "hiding" it. WDYT?

Reply:
ps.: this applies specially for sections like how to setup with automatic or custom certificate management, but not to "How does it work"